"WP API" -> "WordPress REST API" in README files

kadamwhite opened the pull request:
Fixes #2517, based on work by @davetgreen

This only changes content in prose (no change to links or changelog), and varies use of "WordPress REST API", "REST API" and "API" based on context to avoid distracting repetition.

kadamwhite commented:
Looks like my editor changed the line endings on readme.txt or something; is that file auto-generated? I can run a script instead of making the edit in situ

kadamwhite commented on README.md:
```diff
@@ -32,13 +32,9 @@ Update user with ID 4? Send a `POST` request to `/wp-json/wp/v2/users/4`. Get al
-WP API exposes a simple yet easy interface to WP Query, the posts API, post
-meta API, users API, revisions API and many more. Chances are, if you can do
-it with WordPress, WP API will let you do it.
+The WordPress REST API exposes a simple yet easy interface to WP Query, the posts API, post meta API, users API, revisions API and many more. Chances are, if you can do it with WordPress, API will let you do it.
```
Also grammar nit I only noticed in this view, maybe "simple yet powerful"? Simple yet easy seems redundant :)

aaroncampbell replied:
"simple yet powerful" sounds good. Alternatively we could just say "simple" or "easy to use"...but I feel like both of those are questionably accurate by themselves.

rmccue commented:
> Looks like my editor changed the line endings on readme.txt or something; is that file auto-generated? I can run a script instead of making the edit in situ

Hmm, not as far as I know. Can you revert it back?

kadamwhite committed:
"WP API" -> "WordPress REST API" in README files

Fixes #2517, based on work by @davetgreen

This only changes content in prose (no change to links or changelog),
and varies use of "WordPress REST API", "REST API" and "API" based on
context to avoid distracting repetition.
